index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
Looks good only issue is that this `TODO` to make the port configurable will be a bit more tricky. My original thought would be to take a `--port` option on the command line and pass that directly to testem within the command handler. 

But at this point, we do not have access to the command line arguments (at least I don't see that anywhere in the [docs](https://github.com/ember-cli/ember-cli/blob/master/ADDON_HOOKS.md#config) so I think we would have to use the application configuration for the type of build?